pkg/process/analyze.go

```diff
@@ -16,21 +16,18 @@ package process
 
 import (
 	"debug/elf"
-	"debug/gosym"
 	"errors"
 	"fmt"
 	"os"
-	"strings"
 
 	"github.com/hashicorp/go-version"
-	"golang.org/x/arch/x86/x86asm"
 
 	"go.opentelemetry.io/auto/pkg/log"
 	"go.opentelemetry.io/auto/pkg/process/ptrace"
 )
 
 const (
-	mapSize = 15 * 1024 * 1024
+	mapSize = 4096 * 6 * 1024
```

Comment:
Can you comment what these numbers mean?

Reply:
Updated to add description and raised #130 to review size / configuration options.